ToDoWebApi/Applications/UserOperations/Commands/DeleteUser/DeleteUserCommandValidator.cs

```diff
@@ -0,0 +1,13 @@
+using FluentValidation;
+
+namespace ToDoWebApi.Applications.UserOperations.Commands.DeleteUser
+{
+    public class DeleteUserCommandValidator : AbstractValidator<DeleteUserCommand>
+    {
+        public DeleteUserCommandValidator()
+        {
+            RuleFor(command => command.Model.Email).NotEmpty().EmailAddress();
+            RuleFor(command => command.Model.Password).NotEmpty().MinimumLength(6);
+        }
```

Comment:
Senin database'inde id'ler 1'den başlıyordu diye hatırlıyorum, bu yüzden bence buraya Id'nin default değerine, yani 0'a eşit olmaması da eklenebilir.